File: 2021/day-21/main_test.py
import unittest

import main


class TestCase(unittest.TestCase):
    def test_part_one(self):
        self.assertEqual(main.part_one(4, 8), 739785)

    def test_part_two(self):
        self.assertEqual(main.part_two(4, 8), 444356092776315)

File: 2022/day-03/main_test.go
package main

import "testing"

func TestPartOne(t *testing.T) {
	input, err := readLines("input_test.txt")
	if err != nil {
		panic(err)
	}

	got := partOne(input)
	want := 157
	if got != want {
		t.Errorf("got %v, wanted %v", got, want)
	}
}

func TestPartTwo(t *testing.T) {
	input, err := readLines("input_test.txt")
	if err != nil {
		panic(err)
	}

	got := partTwo(input)
	want := 70
	if got != want {
		t.Errorf("got %v, wanted %v", got, want)
	}
}
